docs/design/eth-subscribe.md

```diff
@@ -200,6 +206,15 @@ if (success) {
 
 3. Create a scheduled job that loops through all saved subscriptions and calls the Mirror node Rest API and sends the response to all `subscribers`. It should be possible to configure the time interval. A throttling mechanism can be implemetented that limits the consecutive calls to the Mirror node, and at every interval only X subscriptions with the oldest `lastUpdated` should be polled.
 4. Whenever a connection is closed all corresponding subscriptions should be deleted from memory.
+   
+### Poll for Blocks, `newHeads` Implementation 
+The `newHeads` implementation simply builds on the existing mechanism for polling for logs.  The differences with the polling for logs is as follows:
+1. Add a condition check for `newHeads`, `validateSubscribeEthNewHeads`, and a `newHeads` subscription to the `webSocketServer.ts`.
+2. Add a call to the relay subscribe with the socket, event, and filter. 
+3. Add a call to the mirror node in the `poller.ts` to handle the `newHeads` event by calling `eth_getBlockByNumber` for `latest`.
```

Comment:
Is the logic here poll mirror node for latest block number and then repeat.
How do we ensure there's no missed block?

Reply:
Looking at logs the issue I'm getting at may be the same for both.
Let's see

Reply:
Initial runs didn't show missing or duplicate blocks, and there is a planned test to cover that case, however I could see the behavior changing under load.  Load testing is also planned.  Is there an issue with missing or duplicated logs now?